.github/workflows/ci.yml

```
@@ -91,6 +91,8 @@ jobs:
   build-system-dynamic:
     strategy:
       fail-fast: false
+      matrix:
+        libgit2: [ '1.0.0', '1.0.1', '1.1.0' ]
```

```suggestion
        libgit2: [ '1.1.0' ]
```

we probably only need to have a matrix with at most two versions: the current branch's version (in this case 1.1.0) and the largest known version number (in this case 1.1.0). patch versions only carry bugfixes with no API changes, so we can ignore those.

when this is cherry-picked to other branches (say, v30), it will need to be changed to

```yaml
        libgit2: [ '1.0.0', '1.1.0' ]
```

---

Changed the version matrix.